### PR TITLE
fix(cluster.py): Remove command of yum makecache

### PR DESCRIPTION
**Pull request by yarongilor**
This command fails frequently and is
	not a must to use.
Fixes: #4977 
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
Tested and passed in: https://jenkins.scylladb.com/job/scylla-staging/job/Longevity_yaron/job/artifacts-centos7-test-makecache/

**Comment by fgelcer** [comment]
@yarongilor , please run it another 4 times, just to be sure it won't fail randomly because of missing this command (not sure it will, but let's have more runs to be sure)

File: sdcm/cluster.py
Comment (yarongilor) [inline sdcm/cluster.py:1908]: @fgelcer , i thought of removing this comment, but on second thought, it might be usefull to leave it for a while, to verify no flakiness or other issues pops up in coming future.

File: sdcm/cluster.py
Comment (fgelcer) [inline sdcm/cluster.py:1908]: if you are adding this comment, add the link to the issue

**Comment by yarongilor** [comment]
> @yarongilor , please run it another 4 times, just to be sure it won't fail randomly because of missing this command (not sure it will, but let's have more runs to be sure)

all 4 runs passed.

**Review by fruch — APPROVED** [review]
LGTM

**Comment by fruch** [comment]
@roydahan waiting for you approval for backporting

**Comment by roydahan** [comment]
Do we really need to backport?
Seems like we lived with this issue quite long time and it's not affecting anything.

**Comment by fruch** [comment]
> > Do we really need to backport?
> > Seems like we lived with this issue quite long time and it's not affecting anything.
> 
> i agree, if we will be stuck with 2022.1 we can then backport it, but i don't remember seeing it in 5.0 nor in 2022.1 so far

yaron reported it on 5.0 at least twice according to #4977, this change seems to be harmless.

once we have issue with rpm mirrors, we will have a problem. regardless of this change.

**Comment by yarongilor** [comment]
> > Do we really need to backport?
> > Seems like we lived with this issue quite long time and it's not affecting anything.
> 
> i agree, if we will be stuck with 2022.1 we can then backport it, but i don't remember seeing it in 5.0 nor in 2022.1 so far

@roydahan , @fgelcer , @fruch , this is all not correct:
1. the failures happened only in 5.0 (see in https://jenkins.scylladb.com/job/scylla-5.0/job/artifacts-offline-install/job/artifacts-centos7-test/ )
2. It is not a matter of bad mirrors like it is a matter of a wrong command that shouldn't be used.

**Comment by fruch** [comment]
> > > Do we really need to backport?
> > > Seems like we lived with this issue quite long time and it's not affecting anything.
> > 
> > 
> > i agree, if we will be stuck with 2022.1 we can then backport it, but i don't remember seeing it in 5.0 nor in 2022.1 so far
> 
> @roydahan , @fgelcer , @fruch , this is all not correct:
> 
> 1. the failures happened only in 5.0 (see in https://jenkins.scylladb.com/job/scylla-5.0/job/artifacts-offline-install/job/artifacts-centos7-test/ )
> 2. It is not a matter of bad mirrors like it is a matter of a wrong command that shouldn't be used.

The command is redundant, yes. the reason it was failing was issue with reaching mirrors and not with the command itself. it can happen during any yum install/update command.

**Comment by yarongilor** [comment]
> > > > Do we really need to backport?
> > > > Seems like we lived with this issue quite long time and it's not affecting anything.
> > > 
> > > 
> > > i agree, if we will be stuck with 2022.1 we can then backport it, but i don't remember seeing it in 5.0 nor in 2022.1 so far
> > 
> > 
> > @roydahan , @fgelcer , @fruch , this is all not correct:
> > 
> > 1. the failures happened only in 5.0 (see in https://jenkins.scylladb.com/job/scylla-5.0/job/artifacts-offline-install/job/artifacts-centos7-test/ )
> > 2. It is not a matter of bad mirrors like it is a matter of a wrong command that shouldn't be used.
> 
> The command is redundant, yes. the reason it was failing was issue with reaching mirrors and not with the command itself. it can happen during any yum install/update command.

@fruch , in practice i think we didn't encounter any mirror failure in any other case for any other job. BTW, this could probably worked-around as well by replacing `$releasever` with a specific number in the repo file, but i didn't bother doing it from this exact reason.

**Comment by roydahan** [comment]
The question is, does it fail the test due to this issue or not?
If yes, let’s backport.

**Comment by fgelcer** [comment]
> The question is, does it fail the test due to this issue or not? If yes, let’s backport.

IIRC yes, because it will be stuck until the test times out

**Comment by roydahan** [comment]
@yarongilor please provide the actual answer, and if so, put the backport labels back.

**Comment by yarongilor** [comment]
@roydahan , the test is eventually aborted because this error causes a delay that prevents the test to complete in 30 minutes.
( Details are in https://github.com/scylladb/scylla-cluster-tests/issues/4977 )

**Comment by roydahan** [comment]
ok.

**Comment by fgelcer** [comment]
@fruch @soyacz , please backport

**Pull request merged**
